src/instance.ts

```diff
@@ -23,6 +23,26 @@ import * as extend from 'extend';
 import * as is from 'is';
 import snakeCase = require('lodash.snakecase');
 import {Database} from './database';
+import {SessionPoolOptions, SessionPool} from './session-pool';
+import {Operation as GaxOperation} from 'google-gax';
+import {google as databaseAdmin} from '../proto/spanner_database_admin';
+
+export interface CreateDatabaseOptions {
```

Comment:
This should probably extend [`ICreateDatabaseRequest`](https://github.com/googleapis/nodejs-spanner/blob/master/proto/spanner_database_admin.d.ts#L535)

Reply:
taken care in Instance PR#[619](https://github.com/googleapis/nodejs-spanner/pull/619)